3-enrich/spark-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.spark/EnrichJobSpec.scala

```diff
@@ -243,6 +243,16 @@ object EnrichJobSpec {
                 |"uri": "http://iglucentral.com"
               |}
             |}
+          |},
+          |{
+            |"name": "Ben",
```

Comment:
We're not going to delete for production?

Reply:
it's in the checklist of the pr above to rm it once an iglu central release has gone out

Reply:
Sorry, I just checked if temporary commit exists.